arch/arm/soc/nordic_nrf5/nrf52/soc_power.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SOC_POWER_H_
+#define _SOC_POWER_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*
+ * Bit 0 from GP0 register is used internally by the kernel
+ * to handle PM multicore support. Any change on QMSI and/or
```

Comment:
I don't think we should mention QMSI in a Nordic file?

Reply:
Yes i missing removing this, I copied this portion from Curie, I will remove this,